lua/swampshop/vgui/customizer.lua

```diff
@@ -296,523 +263,538 @@ local function Container(parent, label)
-        XSL = SliderMaker(pane, "Forward/Backward")
-        XSL:SetMinMax(itmcw.pos.min.x, itmcw.pos.max.x)
-        XSL:SetValue(translate.x)
-        YSL = SliderMaker(pane, "Left/Right")
-        YSL:SetMinMax(itmcw.pos.min.y, itmcw.pos.max.z)
-        YSL:SetValue(translate.y)
-        ZSL = SliderMaker(pane, "Up/Down")
-        ZSL:SetMinMax(itmcw.pos.min.z, itmcw.pos.max.z)
-        ZSL:SetValue(translate.z)
-        local pane = Container(wear_container, "Angle")
-        local rotate = (self.item.cfg[self.wear] or {}).ang or (pone and (self.item.wear.pony or {}).rotate) or self.item.wear.rotate
-        XRSL = SliderMaker(pane, "Pitch")
-        XRSL:SetMinMax(-180, 180)
-        XRSL:SetValue(rotate.p)
-        YRSL = SliderMaker(pane, "Yaw")
-        YRSL:SetMinMax(-180, 180)
-        YRSL:SetValue(rotate.y)
-        ZRSL = SliderMaker(pane, "Roll")
-        ZRSL:SetMinMax(-180, 180)
-        ZRSL:SetValue(rotate.r)
-        local pane = Container(wear_container, "Scale")
-        local scale = (self.item.cfg[self.wear] or {}).scale or (pone and (self.item.wear.pony or {}).scale) or self.item.wear.scale
-
-        if isnumber(scale) then
-            scale = Vector(scale, scale, scale)
-        end
+function PANEL:AddImgurOption(section, keys, label)
+    local item = self.item
+    local options = GetNestedInfo(item, keys)
+    local pane = self:AddContainer(section, label)
+    local texturebarhelp = vgui.Create("DLabel", pane)
+    texturebarhelp:Dock(TOP)
+    texturebarhelp:DockMargin(0, 0, 0, SS_COMMONMARGIN)
+    texturebarhelp:SetText("Upload an image to imgur.com and enter the link into the field below.\nFor example: https://imgur.com/a/3AOvcC1\nNo videos or GIFs!")
+    texturebarhelp:SetWrap(true)
+    texturebarhelp:SetTall(128)
+    texturebarhelp:SetAutoStretchVertical(true)
+
+    texturebarhelp.UpdateColours = function(pnl)
+        pnl:SetTextColor(MenuTheme_TX)
+    end
 
-        SXSL = SliderMaker(pane, "Length")
-        SXSL:SetMinMax(itmcw.scale.min.x, itmcw.scale.max.x)
-        SXSL:SetValue(scale.x)
-        SYSL = SliderMaker(pane, "Width")
-        SYSL:SetMinMax(itmcw.scale.min.y, itmcw.scale.max.y)
-        SYSL:SetValue(scale.y)
-        SZSL = SliderMaker(pane, "Height")
-        SZSL:SetMinMax(itmcw.scale.min.z, itmcw.scale.max.z)
-        SZSL:SetValue(scale.z)
-
-        local function transformslidersupdate()
-            self.item.cfg[self.wear] = self.item.cfg[self.wear] or {}
-            self.item.cfg[self.wear].pos = Vector(XSL:GetValue(), YSL:GetValue(), ZSL:GetValue())
-            self.item.cfg[self.wear].ang = Angle(XRSL:GetValue(), YRSL:GetValue(), ZRSL:GetValue())
-            self.item.cfg[self.wear].scale = Vector(SXSL:GetValue(), SYSL:GetValue(), SZSL:GetValue())
-            self:UpdateCfg()
-        end
+    local texturec = vgui.Create("DPanel", pane)
```

Comment:
vgui()

Reply:
vgui() seemingly can't be used on any panel created by a function defined inside a vgui stack without that stack error